### DISCUSSION TITLE
Is it possible to display the name of logged-in user on menu in managed portal?

### DISCUSSION THREAD

**Discussion by Mdshakeelse** [Azure/api-management-developer-portal#2605] [Feature request]
Is it possible to display the name of logged in menu in managed portal?

**Comment by ghost** [comment]
@Mdshakeelse, thank you for opening this issue. We will triage it within the next few business days.

**Comment by Mdshakeelse** [comment]
any updates on this issue

**Comment by jsorohova** [comment]
Hello, @Mdshakeelse, sorry for the late response. No, we currently don't have an opportunity to display username in the menu. For now, we can mark it as a feature request and add it to our backlog.

**Comment by ghost** [comment]
@Mdshakeelse, thank you for requesting this feature.
